protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -215,9 +211,9 @@ func (k Keeper) processStoredPremiums(
 				metrics.EpochInfoName,
 				newEpochInfo.Name,
 			),
-			metrics.GetLabelForIntValue(
-				metrics.EpochNumber,
```

Comment:
Will update this [monitor](https://app.datadoghq.com/monitors/120602974) accordingly after merging